pyspecProcScripts/simple_functions.py

```diff
@@ -1,6 +1,9 @@
 "First order functions for very simple (a few lines) data manipulation"
 import numpy as np
+import pyspecdata as psd
+import logging
 
+logging.basicConfig(format="%(levelname)s:%(message)s", level=logging.INFO)
```

Comment:
If I just used logging the message wouldn't show up in the terminal (even if I used psd.strm) so I follow [this](https://docs.python.org/3/howto/logging.html)

Reply:
you don't want this here.  Logging will not show up unless you turn it on *in the script*.  So whatever example you have should have a `psd.init_logging()` at the top.

Reply:
btw `psd.init_logging` includes something like the config, and particularly points the log file to a particular location.

The reason why we prefer logging over printing for anything that gets committed, is that it's much easier to read through the log to figure out what's up vs. looking on the command line.

Reply:
along those lines, I would actually make the statement that says everything is  good a debug, not info --> we don't need to see that everything is OK, only be alerted if something is funky